Makefile

```diff
@@ -57,6 +57,11 @@ deps:
 linter:
 	$(GO_LINT) version || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GO_PATH)/bin v1.50.0
 
+##@ General
+
+.PHONY: generate-proto
+generate-proto: ##generate data collect proto
```

Comment:
This command should be invoke before the plugin or E2E test? Please make sure the all the test can be executed successfully